src/modules/plotNFT.ts

```diff
@@ -120,11 +120,11 @@ export function pwAbsorbRewards(walletId: number, fee?: string) {
   };
 }
 
-export function pwSelfPool(walletId: number) {
+export function pwSelfPool(walletId: number, fee?: number) {
   return async (dispatch) => {
     const { data } = await async_api(
       dispatch,
-      pwSelfPoolMessage(walletId),
+      pwSelfPoolMessage(walletId, fee),
```

Comment:
Since `fee?` is an optional, is there any issue with `pwSelfPoolMessage()` calling `format_message()` with a potentially undefined value?